setup.py

```diff
@@ -78,11 +78,8 @@
         "Operating System :: OS Independent",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12",
```

Comment:
we are running tests for 3.8, 3.9, 3.10, 3.11 and 3.12, i think you can remove 3.6 and 3.7 from the list